README.md

```diff
@@ -131,6 +131,16 @@ To run the SDK tests, simply run from the root of this repository:
 pnpm test
 ```
 
+## Troubleshooting
+
+If you see import error when you do this
+
+```javascript
```

Comment:
Nit I think this should be typescript since it only applies to typescript

Reply:
fixed